index.html

```diff
@@ -8,129 +8,34 @@
-          </div>
-
-        <!--- Headline Section --->
-          <div class="headline">
-            <div class="headline-text1">Hello, I am Ben.</div>
-            <div class="headline-text2">The fullstack developer you didn't know you needed.</div>
-            <div class="headline-details">Discover my work and contact me to reach the sky together</div>
-            <div class="social">
-              <div class="let-connect">LET’S CONNECT</div>
-
-              <!-- Add font awesome icons -->
-              <ul class="icon-list">
-                <li><a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a></li>
-                <li><a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a></li>
-                <li><a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a></li>
-                <li><a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a></li>
-                <li><a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
-              </ul>
-            </div>
-          </div>
+  <!-- Header Section  -->
+    <section class="header">
+      <!--- Top navigation Bar --->
+      <div class="topnav">
+        <a href="#default" class="logo">My Logo</a>
+        <div class="nav-menu" id="myLinks">
+          <a href="#portfolio">Portfolio</a>
+          <a href="#about">About</a>
+          <a href="#contact">Contact</a>
         </div>
+        <!-- "Hamburger menu" / "Bar icon" to toggle the navigation links -->
+        <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+          <i class="fa fa-bars"></i>
+        </a>
       </div>
 
-      <!-- The "works" categories are here -->
-      <section class="grid-container">
-        <section class="grid-item">
-            <img class="work-section-image" src="ressources/Work-1.png" alt="Picture for the first work of Benjamin Patin">
-            <h2>Tonic</h2>
-            <ul class="client-info">
-              <li class="client-name">CANOPY</li>
-              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
-              <li class="project-style">Back End Dev</li>
-              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
-              <li class="project-year">2015</li>
-            </ul>
-            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="language-used">
-              <li><div class="tags">html</div></li>
-              <li><div class="tags">css</div></li>
-              <li><div class="tags">javaScript</div></li>
-            </ul>
-          <button type="button" class="button">See Project!</button>
-        </section>
-        <section class="grid-item">
```

Comment:
Projects on even numbers as 2 and 4 should have a different design than 1 and 3. Kindly see the screenshot below
![2](https://user-images.githubusercontent.com/35504149/139494875-c1f2d17a-0856-4c4f-85b3-8f31c98d4bd7.png)
![3](https://user-images.githubusercontent.com/35504149/139494925-43f65a88-b852-49f8-9ad9-641f06b9d4ea.png)

Reply:
I have been trying all day to get to do it, any suggestions ?